### PR TITLE
Clear lines on goal, log speed, formatting, increase speed and turn radius over time

### PR DESCRIPTION
**Pull request by theowiik**
- Clear lines on goal
- log speed
- formatting
- increase speed and turn radius over time

### CODE REVIEW

File: weave/Scripts/Main.cs [hunk 1]
```diff
@@ -50,6 +53,16 @@ public override void _Ready()
         SetupLogger();
     }
 
+    public override void _Process(double delta)
+    {
+        _players.ForEach(
```
Comment (erikwessman) [inline weave/Scripts/Main.cs:58]: Should discuss this before merging ☺️

Comment (theowiik) [reply weave/Scripts/Main.cs]: You're thinking about good values? Or what its based on

Comment (erikwessman) [reply weave/Scripts/Main.cs]: Perhaps both

Comment (erikwessman) [reply weave/Scripts/Main.cs]: 😊

**Review by erikwessman — APPROVED** [review]

**Review by cvazouras — APPROVED** [review]

**Pull request merged**